src/Screens/Profile/index.tsx

```diff
@@ -45,7 +45,7 @@ const Profile: FunctionComponent<ProfileProps> = ({ navigation }) => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
   const user = useAppSelector(selectUser);
-  const diveLogs = Object.values(useAppSelector(selectAllDiveLogs));
+  const diveLogs = useAppSelector(selectOrderedDiveLogs);
```

Comment:
this is what messed up the ordering, need to save the proper order from the backend